### ISSUE TITLE
Feature request: Import existing project

### ISSUE THREAD

**Issue by hristo-vrigazov** [codenameone/CodenameOne#2084]
There must be an easy way to import an existing project. I recently had to switch the PC I developed on, and I had to recreate the whole project, copy all the themes file, add all the CN1libs I used, etc. I cannot believe something similar has not been requested before.

**Comment by amiga** [comment]
I agree, but you can import your .res into the gui builder, it was awkward on old one, but it works. Not sure about new one :)

**Comment by shannah** [comment]
I transfer projects between machines all the time.  You don't need to import them, you should just be able to "open" them as long as you're not changing to a different IDE (e.g. Netbeans to Eclipse).
